cmd/buildah/run.go

```diff
@@ -79,10 +79,12 @@ func runCmd(c *cli.Context) error {
 		Args:     c.StringSlice("runtime-flag"),
 	}
 
-	if c.Bool("tty") {
+	options.Terminal = buildah.DefaultTerminal
+	if c.IsSet("tty") {
```

Comment:
Does this path get taken when we're passed `--tty=false`?

Reply:
I'd not specifically watched this code, but the top level run code was only getting options.Terminal=buildah.WithoutTerminal when I set the flag to false.  I'll instrument this code during my testing to verify.